src/gp_doctest.cpp

```diff
@@ -573,10 +565,10 @@ TEST_CASE("GPInstance: GenerateCompleteRootedTreeCollection") {
   inst.GetEngine()->SetBranchLengths(branch_lengths);
   // Because the branch lengths contain the GPCSP index, we can check that the indices
   // correspond to what we see in the GPCSP DAG in
-  // https://github.com/phylovi/bito/issues/349#issuecomment-897233859
+  // https://github.com/phylovi/bito/issues/391#issuecomment-1168048090
   CHECK_EQ(inst.GenerateCompleteRootedTreeCollection().Newick(),
-           "((0:7,1:9):3,(2:11,(3:12,4:13):5):2):0;\n"
-           "(1:10,(0:8,(2:11,(3:12,4:13):5):6):4):0;\n");
+           "((0:7,1:9):3,(2:11,(3:12,4:13):2):6):0;\n"
+           "(1:10,(0:8,(2:11,(3:12,4:13):2):5):4):0;\n");
```

Comment:
Just curious, why were these branch lengths changed?

Reply:
I don't think the branch lengths actually changed. Those numbers refer to the edge idxs above that node which were changed when PCSPCounter became an ordered map.